presto-main/src/main/java/com/facebook/presto/sql/planner/plan/Assignments.java

```diff
@@ -164,6 +164,27 @@ public int size()
         return assignments.size();
     }
 
+    @Override
+    public boolean equals(Object o)
+    {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        Assignments that = (Assignments) o;
+
+        return assignments.equals(that.assignments);
+    }
+
+    @Override
+    public int hashCode()
```

Comment:
Why? I'm not aware of such a convention, in fact there's no consistency in Presto in this regard